.github/pull_request_template.md

```diff
@@ -2,12 +2,22 @@
 
 _**Please read the [CONTRIBUTING.md](https://github.com/MaibornWolff/codecharta/blob/main/CONTRIBUTING.md) before opening a PR.**_
 
-Issue: #
+{Issue/Closes}: #
 
 ## Description
 
 **Descriptive pull request text**, answering:
   - What problem/issue are you fixing?
   - What does this PR implement and how?
 
+## Definition of Done
+
+A PR is only ready for review once all the following acceptance criteria are fulfilled:
```

Comment:
I would not forbid to have a PR so that anyone can get review as soon as possible from others.

```suggestion
A PR is only ready for merge once all the following acceptance criteria are fulfilled:
```